settings.php

```diff
@@ -41,4 +41,8 @@
     $settings->add(new admin_setting_configcheckbox('logstore_xapi/backgroundmode',
         get_string('backgroundmode', 'logstore_xapi'),
         get_string('backgroundmode_desc', 'logstore_xapi'), 0));
+
+    $settings->add(new admin_setting_configtext('logstore_xapi/maxbatchsize',
```

Comment:
My thinking here @garemoko, is that on a load balanced LRS, sending lots of requests with a reasonable amount of data will actually be quicker than sending a huge number of statements.